scripts/reset_invalid_store_data.py

```diff
@@ -0,0 +1,94 @@
+import os
+import sys
+
+parent_dir_path = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
+sys.path.append(parent_dir_path)
+
+from sqlalchemy import create_engine
+from sqlalchemy.exc import SQLAlchemyError, IntegrityError
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy import Column
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy.types import Boolean
+from sqlalchemy.dialects.postgresql import JSONB, UUID, TIMESTAMP
+
+import settings
+
+try:
+    db = create_engine(settings.DB_URI)
+    Session = sessionmaker(db)
+    session = Session()
+    base = declarative_base()
+except SQLAlchemyError as e:
+    print(e)
+    raise
+
+
+class SurveyResponse(base):
+    __tablename__ = 'responses'
+    tx_id = Column("tx_id",
+                   UUID,
+                   primary_key=True)
+
+    ts = Column("ts",
+                TIMESTAMP(timezone=True))
+
+    invalid = Column("invalid",
+                     Boolean,
+                     default=False)
+
+    data = Column("data", JSONB)
+
+    def __init__(self, tx_id, invalid, data):
+        self.tx_id = tx_id
+        self.invalid = invalid
+        self.data = data
+
+    def __repr__(self):
+        return '<SurveyResponse {}>'.format(self.tx_id)
+
+    def to_dict(self):
+        return {c.name: getattr(self, c.name) for c in self.__table__.columns}
```

Comment:
This is duplicated from server.py. Have we considered extending the store api to allow us to update or overwrite an existing stored submission? Then a script would only need to interact with the API and not need any database code.

Reply:
It's considered and there's even a card to do the work.  Because this is fixing an ops issue I think getting it fixed is the important thing, and the refactoring can some separately

Reply:
and would we want to modify a submission via the API?  Having such a thing would imply that it's something we support and changing something we stored should be very exceptional

Reply:
@ajmaddaford I don't like this PR myself and yep extending the store api would be better. However I felt that's a part of a wider improvement to sdx-store. Included in that wider change is the fact I also don't like that this `invalid` key is being passed in the response data. To me that should always stay untouched. I would much rather modify collect to pass the invalid status in the post.

Reply:
I would also like to review how we reprocess. At the moment we don't drop it at the start of the SDX system. In my opinion it should drop it at the very start and it's revalidated etc. This would make all this work mute.

Reply:
> It's considered and there's even a card to do the work. Because this is fixing an ops issue I think getting it fixed is the important thing, and the refactoring can some separately

I don't think adding a new endpoint for replacing a stored submission would have taken much longer than what's been done already?

> and would we want to modify a submission via the API? Having such a thing would imply that it's something we support and changing something we stored should be very exceptional

If we're doing it in a script, we're supporting it, however exceptional. Doing it via the API would ensure consistency and if necessary we could audit who/what made the changes and when.